modules/installation-user-infra-exporting-common-variables-arm-templates.adoc

```diff
@@ -40,41 +40,69 @@ provided ARM templates:
 +
 [source,terminal]
 ----
-$ export CLUSTER_NAME=<cluster_name><1>
-$ export AZURE_REGION=<azure_region><2>
-$ export SSH_KEY=<ssh_key><3>
-$ export BASE_DOMAIN=<base_domain><4>
-$ export BASE_DOMAIN_RESOURCE_GROUP=<base_domain_resource_group><5>
+$ export CLUSTER_NAME=<cluster_name>
+----
+* `<cluster_name>`: The value of the `.metadata.name` attribute from the `install-config.yaml` file.
```

Comment:
Per OSDOCS guidelines, alternative to numbered callouts is a description list in [this format](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks).This applies to all the callout instances in this PR.